src/viam/sdk/module/service.cpp

```diff
@@ -1,3 +1,9 @@
+#ifdef _WIN32
+#define NOMINMAX
+#include <winsock2.h>
```

Comment:
I don't love this. I'm surprised the gRPC headers don't handle this for themselves, but without this this file doesn't compile.

The right way to do this would be with a global config precompiled header that got pulled into every `.cpp` we have, but I don't want to spend time on that right now, so I will file a follow-up ticket.

Reply:
Filed https://viam.atlassian.net/browse/RSDK-10389